src/Admin/Admin.php

```diff
@@ -230,4 +238,59 @@ public function get_view( string $view, array $context_variables = [] ): string
 	protected function enableReports(): bool {
 		return apply_filters( 'woocommerce_gla_enable_reports', true );
 	}
+
+	/**
+	 * This method is ONLY used during development.
+	 *
+	 * The runtime.js file is created when the front-end is developed in Fast Refresh mode
+	 * and must be loaded together to enable the mode.
+	 *
+	 * When Gutenberg is not installed or not activated, the react dependency will not have
+	 * the 'wp-react-refresh-entry' handle, so here injects the Fast Refresh scripts we built.
+	 *
+	 * The Fast Refresh also needs the development version of React and ReactDOM.
+	 * They will be replaced if the SCRIPT_DEBUG flag is not enabled.
+	 *
+	 * @param WP_Scripts $scripts WP_Scripts instance.
+	 */
+	private function inject_fast_refresh_for_dev( $scripts ) {
+		if ( ! file_exists( "{$this->get_root_dir()}/js/build/runtime.js" ) ) {
+			return;
+		}
+
+		$react_script = $scripts->query( 'react', 'registered' );
+
+		if ( ! $react_script ) {
```

Comment:
❓  Shouldn't be also react-dom a condition to abort?

Reply:
Assumed they are always used together in the webpage scope, so I think it should be ok to only check one of them. Ref: https://github.com/WordPress/wordpress-develop/blob/5.6.8/src/wp-includes/script-loader.php#L83-L84